pattern-1/bosh-release/config/blobs.yml

```diff
@@ -0,0 +1,20 @@
+jdbcdrivers/mssql-jdbc-7.0.0.jre8.jar:
```

Comment:
Shall we empty this file as discussed offline?

Reply:
fixed https://github.com/wso2/pivotal-cf-apim/pull/52/commits/35c6554ff64c8f44c531e2ed5ac7534cb6a2717d